server/server.js

```diff
@@ -34,10 +34,10 @@ require('./config/passport')(passport)
 app.use(passport.initialize());
 app.use(passport.session());
 
-app.get('/login/auth/google', passport.authenticate('google', { scope: ['profile'] }))
+app.get('/login/auth/google', passport.authenticate('google', { scope: ['profile', 'email'] }))
 
-app.get('/login/auth/google/callback', passport.authenticate('google', { failureDirect: '/'}), (req, res) => {
-    res.redirect('/');
+app.get('/login/auth/google/callback', passport.authenticate('google', { failureDirect: '/', successRedirect: '/'}), (req, res) => {
```

Comment:
Do we need a final (req,res) middleware? Or can passport.authenticate be last function?